cloudformation/bulkExport.yaml

```diff
@@ -166,7 +168,18 @@ Resources:
     Type: AWS::Glue::Job # TODO: update this configuration once we have an actual ETL script
     Properties:
       Role: !GetAtt GlueJobRole.Arn
+      GlueVersion: '2.0'
+      WorkerType: 'G.2X'

```

Comment:
In testing for exporting 1 GB of data from DDB, I found that this `G.2X` workers resulted in the fastest export time for the number of DPU used. Our export task is memory intensive and G2 workers have the most memory.